frameworks/cassandra/src/main/dist/svc.yml

```diff
@@ -103,6 +103,14 @@ pods:
         goal: FINISHED
         cmd: aws s3 cp container-path/snapshot/ s3://${S3_BUCKET_NAME}/${SNAPSHOT_NAME}/node-${POD_INSTANCE_INDEX}/ --recursive
         resource-set: sidecar-resources
+      upload-azure:
+        goal: FINISHED
+        cmd: >
+                az login -u $CLIENT_ID --service-principal --tenant $TENANT_ID -p $CLIENT_SECRET
```

Comment:
I'm sure this is correct, but I'll still ask. Does `--service-principal` takes an argument like other flags in this command ?